src/atomate2/common/utils.py

```diff
@@ -11,6 +11,65 @@
 if TYPE_CHECKING:
     from pathlib import Path
 
+from pymatgen.io.phonopy import get_phonopy_structure
```

Comment:
I think this should be fixed in pymatgen directly. @janosh what do you think?

Reply:
Agreed. PR would be great 👍